README.md

```diff
@@ -65,16 +67,11 @@ spec:
           git: git@github.com:crafting-dev/template-ruby-rails.git
```

Comment:
For open source repos, please use `https://github.com/crafting-dev/template-ruby-rails.git` otherwise, non-member will have problem checking out the code.

Reply:
Good point, updated in b204a4c.